src/Glue.elm

```diff
@@ -351,8 +294,18 @@ viewSimple (Glue rec) v msg model =
 -- Helpers
 
 
-{-| Tiny abstraction over [`Cmd.map`](https://package.elm-lang.org/packages/elm/core/latest/Platform-Cmd#map)
+{-| A tiny abstraction over [`Cmd.map`](https://package.elm-lang.org/packages/elm/core/latest/Platform-Cmd#map)
 packed in `(model, Cmd msg)`.
+
+    -- Parent module
+    type Msg
+        = ChildMsg Child.Msg
+
+    update =
+        -- ...
+        childModelCmdPair
+            |> Glue.map ChildMsg
+
```

Comment:
Does this example make sense?

Reply:
probably an example usage might be as part of `updateWith` in the case where say function returns child msg but glue is defined using poly it can be patched using `map`